adage/requirements.txt

```diff
@@ -1,7 +1,7 @@
 argparse==1.2.1
 Django==1.8.11
 django-fixtureless==1.4.3.3
-django-haystack==2.4.1
+django-haystack==2.6.0
```

Comment:
Note - this newer django-haystack version is needed in order to work with Elasticsearch v2.x